### PR TITLE
Add Ocean's Logger and use it

### PR DESCRIPTION
**Pull request by Geod24**
Nothing to see here

### CODE REVIEW

File: source/agora/utils/Log.d [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+}
+
+/// Convenience alias
+public alias LogLevel = Level;
+
+/// Initialize the logger
+static this ()
+{
+    import ocean.util.log.Logger;
```
Comment (AndrejMitrovic) [inline source/agora/utils/Log.d]: This import already exist in module scope.

File: source/ocean/util/log/Logger.d [hunk 2]
```diff
@@ -782,7 +782,7 @@ public final class Logger : ILogger
 
     ***************************************************************************/
 
-    public Logger append (Level level, lazy cstring exp)
+    public Logger append (Level level, lazy cstring exp) @trusted
```
Comment (AndrejMitrovic) [inline source/ocean/util/log/Logger.d]: Commit message says nothrow but I don't see it here.

**Comment by AndrejMitrovic** [comment]
Ok just kill the BanManager commit and it will go green?

**Comment by Geod24** [comment]
> Ok just kill the BanManager commit and it will go green?

I had to remove an unused function because the C symbol it references does not exist on Musl...
And one allocation test fails, let me look into it.

**Comment by Geod24** [comment]
Updated, now it's less than 200 LoC!

File: source/agora/consensus/data/UTXOSet.d [hunk 3]
```diff
@@ -188,15 +189,15 @@ private class UTXODB
     {
         const db_exists = utxo_db_path.exists;
         if (db_exists)
-            logInfo("Loading UTXO database from: %s", utxo_db_path);
+            log.info("Loading UTXO database from: %s", utxo_db_path);
```
Comment (AndrejMitrovic) [inline source/agora/consensus/data/UTXOSet.d]: `%s` => `{}`

File: source/agora/consensus/data/UTXOSet.d [hunk 4]
```diff
@@ -188,15 +189,15 @@ private class UTXODB
...
 
         // todo: can fail. we would have to recover by either:
         // A) reconstructing it from our blockchain storage
         // B) requesting the UTXO set from our peers
         this.db = Database(utxo_db_path);
 
         if (db_exists)
-            logInfo("Loaded database from: %s", utxo_db_path);
+            log.info("Loaded database from: %s", utxo_db_path);
```
Comment (AndrejMitrovic) [inline source/agora/consensus/data/UTXOSet.d]: `%s` => `{}`

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**